README.md

```diff
@@ -30,8 +30,8 @@ Catch can report the tests results in [JUnit][junit] formatted xml when enabled,
 
 This file is parsed with Python and the [junitparser][junitparser-lib] library in the `process.py` script that outputs a `results.json` file that respects the test runners specifications.
 
-[test-runner-interface]: https://github.com/exercism/automated-tests/blob/master/docs/interface.md
-[test-runner-docker]: https://github.com/exercism/automated-tests/blob/master/docs/docker.md
+[test-runner-interface]: https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md
+[test-runner-docker]: https://github.com/exercism/docs/blob/main/building/tooling/docker.md
```

Comment:
```suggestion
[test-runner-docker]: https://exercism.org/docs/building/tooling/test-runners/docker
```

Reply:
Thanks for these suggestions Erik, I've pushed a new commit with the changes.